interoperator/pkg/constants/constants.go

```diff
@@ -28,7 +28,7 @@ const (
 
 	NamespaceLabelKey = "OWNER_INTEROPERATOR_NAMESPACE"
 
-	MultiClusterWatchTimeout = 28800 // 8 hours in seconds
+	MultiClusterWatchTimeout = time.Hour * 24
```

Comment:
I think the `time.Hour` resolution is on Nanoseconds. In that case the value will be  `3600000000000`.I go this impression from https://pkg.go.dev/time#Hour.

Reply:
Updated